tests/data/README.rst

```diff
@@ -0,0 +1,5 @@
+Particle test data folder contents
+==================================
+
+This folder contains files used for testing purposes.
+By construction files can have features, hence not be fully correct.
```

Comment:
What is this line trying to say?

Reply:
Really only that in case somebody looks at the files for examples. Better make it clear that they are not examples to follow. The file is not super important but I thought it does not harm.